### PR TITLE
Update: support axum v0.6.

### PR DESCRIPTION
**Pull request by ya7010**

### CODE REVIEW

File: examples/main.rs [hunk 1]
```diff
@@ -22,8 +22,8 @@ struct Data {
 }
 
 async fn post_data(
-    Json(payload): Json<Data>,
     Extension(state): Extension<Arc<Mutex<State>>>,
+    Json(payload): Json<Data>,
```
Comment (lazear) [inline examples/main.rs:26]: Is there a reason for this change?

**Comment by yassun4dev** [comment]
I too find it a bit odd, but changing the position of the argument fixes the compile error.

image

File: examples/main.rs (hunk 1 above)
Comment (ya7010) [reply examples/main.rs:26]: It's too strange, but changing the position of the argument fixes the compile error.

image

Comment (lazear) [reply examples/main.rs:26]: Huh - yeah, I just went through and upgraded a couple projects to axum 0.6 today and noticed this as well. Thanks!

**Pull request merged**